man1/aur.1

```diff
@@ -35,6 +35,18 @@ A different path may be set at build time through the
 macro, for example
 .IR /usr/local/lib/aurutils:/usr/lib/aurutils .
 
+Output colorization can be controlled by setting the environment variable
```

Comment:
I would move this section to an `ENVIRONMENT` section, compare `man-pages(7)`

Reply:
Will change.